src/contracts/Staking.sol

```diff
@@ -100,11 +100,21 @@ interface ITokePool {
     function balanceOf(address owner) external view returns (uint256);
 }
 
+interface ITokeManager {
```

Comment:
I wold probably stick this in its own file under src/interfaces/ITokeManager.sol and import it.

Reply:
Gonna add these in the claim PR because there's some changes around them